lib/spreadsheet_writer.rb

```diff
@@ -1,8 +1,9 @@
 require 'google_drive'
 module Spreadsheet
   class Writer
-    def initialize(spreadsheet_id)
+    def initialize(spreadsheet_id, worksheet_pos = 0)
```

Comment:
Setting the default to `0` seems risky if that equates to the first tab which we are actually using for real data.

Reply:
good point. I changed the value to 99 which should raise a descriptive error from the service indicating the spreadsheet does not have a worksheet at that position.
https://github.com/dxw/teacher-vacancy-service/pull/638/commits/8c08e94d418810a5215591772eb139f59163189e